packages/contractkit/src/wrappers/Governance.ts

```diff
@@ -72,4 +136,298 @@ export class GovernanceWrapper extends BaseWrapper<Governance> {
       stageDurations: res[4],
     }
   }
+
+  getProposalMetadata: (proposalID: BigNumber) => Promise<ProposalMetadata> = proxyCall(
+    this.contract.methods.getProposal,
+    tupleParser(parseNumber),
+    (res) => ({
+      proposer: res[0],
+      deposit: toBigNumber(res[1]),
+      timestamp: toBigNumber(res[2]),
+      transactionCount: toBigNumber(res[3]).toNumber(),
+    })
+  )
+
+  getProposalTransaction: (
+    proposalID: BigNumber,
+    txIndex: number
+  ) => Promise<Transaction> = proxyCall(
+    this.contract.methods.getProposalTransaction,
+    tupleParser(parseNumber, parseNumber),
+    (res) => ({
+      value: toBigNumber(res[0]),
+      destination: res[1],
+      // @ts-ignore string[] bytes type
+      data: toBuffer(res[2]),
```

Comment:
i have a 

```ts
export function parseBytes(input: string): Array<string | number[]> {
  return input as any
}
```
 we can have a `toBuffer` to accept string[], instead of the ts-ignore. it's a common thing this ts-ignore, we should encapsulate it in one place.